lib/post_revisor.rb

```diff
@@ -662,14 +662,19 @@ def bump_topic
   end
 
   def bypass_bump?
-    !@post_successfully_saved || post_changes.any? || @topic_changes.errored? ||
-      @opts[:bypass_bump] == true || @post.whisper? || only_hidden_tags_changed?
+    return true if @opts[:bypass_bump] == true
+    return true if @post.whisper? || !@post_successfully_saved || post_changes.any?
+    return true if @topic_changes.errored?
+    return true if topic_title_changed? || topic_category_changed? || topic_tags_changed?
+    return true if only_hidden_tags_changed?
+
+    false
```

Comment:
I'm probably misunderstanding, but are we ever reaching this return?

Reply:
Actually maybe we should just have no default and only have very  explicit conditions...since there are so many different conditions on bumping